### PR TITLE
Feature/289717 - creating logs for neflix redirection problem

### PR DESCRIPTION
**Pull request by mhnakashima**

### CODE REVIEW

**Review by EricJangola — APPROVED** [review]

**Review by caiopsouza — APPROVED** [review]

**Review by rafael-santiago-take — APPROVED** [review]

**Review by jujucasseb — APPROVED** [review]

File: src/Take.Blip.Builder/FlowManager.cs [hunk 1]
```diff
@@ -409,7 +410,16 @@ private async Task ProcessActionsAsync(LazyInput lazyInput, IContext context, Ac
                         {
                             actionTrace.ParsedSettings = settings;
                         }
-                        
+
+                        if (action is RedirectAction)
```
Comment (andrebires) [inline src/Take.Blip.Builder/FlowManager.cs:414]: The FlowManager should not know specifics for its actions. This logic should not be here.

File: src/Take.Blip.Builder/FlowManager.cs [hunk 2]
```diff
@@ -409,7 +410,16 @@ private async Task ProcessActionsAsync(LazyInput lazyInput, IContext context, Ac
...
+                        {
+                            var redirect = settings.ToObject<Redirect>(LimeSerializerContainer.Serializer);
+                            if (redirect.Address == "netflix-recomendacao")
+                            {
+                                _logger.Warning($"#lognetflix# ({lazyInput.Message}) - ({redirect.Address}) - ({redirect.Context})");
```
Comment (andrebires) [inline src/Take.Blip.Builder/FlowManager.cs:419]: This is a public repository, we should never put information about our customers here.

Comment (andrebires) [reply src/Take.Blip.Builder/FlowManager.cs:419]: And even it was public, we do not do specific things for specific clients.

Comment (mhnakashima) [reply src/Take.Blip.Builder/FlowManager.cs:419]: I'll be removed.

**Pull request merged**